src/controllers/router-store/test.tsx

```diff
@@ -548,12 +548,12 @@ describe('RouterStore', () => {
         const RouteName = () => <>{useRouteName()}</>;
 
         const route = {
-          component: () => null,
+          component: () => <p>home</p>,
```

Comment:
Any particular reason why this was changed?

Reply:
The approach to testing is very different in RTL vs enzyme. The biggest being "test from the perspective of a user". I had to change the test to work by checking for what was rendered.